stdlib/task/src/main/ballerina/src/task/listener.bal

```diff
@@ -46,7 +46,7 @@ public type Listener object {
     #
     # + s - Service to attach to the listener
     # + name - Name of the service
-    # + return - Returns nil if attaching the service is successful
+    # + return - () if attaching the service is successful
```

Comment:
```suggestion
    # + return - () or else a `error` upon failure to register the listener
```

Reply:
Here the error result is panicked. The error will not be returned. Should the description still be changed?